GAE/src/com/gallatinsystems/surveyal/dao/SurveyedLocaleDao.java

```diff
@@ -289,8 +291,16 @@ public SurveyedLocaleDao() {
                 "String", identifier, paramMap);
         appendNonNullParam("displayName", filterString, paramString,
                 "String", displayName, paramMap);
+        if (displayNamePrefix != null && !"".equals(displayNamePrefix)) {
+            String endString = displayNamePrefix + Character.MAX_VALUE;
+            appendNonNullParam("displayName", filterString, paramString, "String", displayNamePrefix, paramMap, GTE_OP);
+            appendNonNullParam("displayName", filterString, paramString, "String", endString, paramMap, LT_OP);
```

Comment:
If we're filtering by a display name _prefix_, I fail to understand why we need to add this extra filter clause :sweat: 
What is the relationship between the operators `<` `>=` and the content of the string? Have we tried this with displayName containing non ASCII characters?

Reply:
Otherwise we do not filter by prefix. Just `>= 'ABC'` will hit ABC*, ABD*...ABZ* and beyond, whereas `>= ABC && < ABC&#xffffffff;` willl hit exactly the strings starting with ABC. And there are hyphens in the test datastore and I tried filtering by those...

Reply:
I could add some äöñ in a test datastore if you think it is weird.

Reply:
Tests ok with accented and cyrillic characters.